.github/workflows/go-postsubmit.yml

```diff
@@ -0,0 +1,40 @@
+name: GoPostSubmit
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
+  workflow_dispatch: {}
+
+env:
+  # Common versions
+  GO_VERSION: '1.16'
+  GO_REQUIRED_MIN_VERSION: ''
+  GOPATH: '/home/runner/work/governance-policy-propagator/governance-policy-propagator/go'
+defaults:
+  run:
+    working-directory: go/src/open-cluster-management.io/governance-policy-propagator
+
+jobs:
+  verify:
+    name: images
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          path: go/src/open-cluster-management.io/governance-policy-propagator
+      - name: install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: install imagebuilder
+        run: go install github.com/openshift/imagebuilder/cmd/imagebuilder@v1.2.1
+      - name: images
+        run: make images
```

Comment:
For our Makefile, this should be `make build-images`. And actually we can update the Makefile to default to `TAG ?= edge` instead of `TAG ?= latest`.

https://github.com/open-cluster-management-io/governance-policy-propagator/blob/2f29465c22e5f8c0ddc74b2b556e64968090cb86/Makefile#L144-L146

Reply:
Nice!  Fixed this.